pkg/reconciler/revision/resources/deploy.go

```diff
@@ -133,15 +161,32 @@ func makePodSpec(rev *v1.Revision, cfg *config.Config) (*corev1.PodSpec, error)
 	}
 
 	var extraVolumes []corev1.Volume
+
+	podInfoFeature, podInfoExists := rev.Annotations[PodInfoFeatureKey]
```

Comment:
podInfo is a bit confusing to me, is there a more descriptive name we could use? (sorry if this was already discussed elsewhere, I'm blanking on it if it was)

Reply:
I could not find a better name - this name should fit all over: E.g.
```
        tokenVolumeName   = "knative-token-volume"
        podInfoVolumeName = "pod-info"
	certVolumeName    = "server-certs"
	logVolumeName     = "knative-var-log"
```

The volume name where we place the .... token, the cert, the podInfo, the log
So podInfo makes sense to me, but will always be glad to change to something better if a candidate comes a long :)